### ISSUE TITLE
add class after render on first section

### ISSUE THREAD

**Issue by patlux** [alvarotrigo/fullPage.js#1063]
The library does not add a "fp-viewing-" class to the body when the page loads the first section.

Workaround for me was to add this class in the "afterRender"-function.

```javascript
afterRender: () ->
  document.body.className += 'fp-viewing-home'
```

**Comment by alvarotrigo** [comment]
Yep, to solve in the next release! Thanks for reporting it!

**Comment by alvarotrigo** [comment]
Solved in 2.6.0.
